woocommerce/woocommerce
# Add product tour to new product management experience

### All Submissions:






### Changes proposed in this Pull Request:

Adds the product tour modal and tour kit to the new product experience.

<img width="444" alt="Screen Shot 2023-01-12 at 5 34 43 PM" src="https://user-images.githubusercontent.com/10561050/212407264-b226f369-0111-481e-9242-cfd8ce62bbf7.png">

<img width="413" alt="Screen Shot 2023-01-13 at 10 06 23 AM" src="https://user-images.githubusercontent.com/10561050/212407295-e09f15ca-1968-4361-9a34-96c6013d123f.png">

Closes (part of) #36322  .






### How to test the changes in this Pull Request:



1. Navigate to the new product experience
2. See the modal appear after a few moments
3. Click the button to begin the tour
4. Walk through the tour, noting that copy and tooltip placement is correct
5. After completing the tour, delete `woocommerce_product_tour_modal_hidden` from your `wp_options` table
6. Reload the page and exit the tour early, making sure that the modal is dismissed
7. Make sure refreshing the page persists the hidden modal and that it does not appear again on page load



### Other information:




### FOR PR REVIEWER ONLY:
